utils/gen-docker-image.sh

```diff
@@ -7,12 +7,26 @@ VER=$(grep -Po '(?<=^__version__ = ).*' $thisd/../heudiconv/info.py | sed 's/"//
 
 image="kaczmarj/neurodocker:0.9.1"
 
-docker run --rm $image generate docker -b neurodebian:bullseye -p apt \
-    --dcm2niix version=v1.0.20220720 method=source \
-    --install git gcc pigz liblzma-dev libc-dev git-annex-standalone netbase \
+docker run --rm $image generate docker \
+    --base-image neurodebian:bullseye \
+    --pkg-manager apt \
+    --dcm2niix \
+        version=v1.0.20220720 \
+        method=source \
+        cmake_opts="-DZLIB_IMPLEMENTATION=Cloudflare -DUSE_JPEGLS=ON -DUSE_OPENJPEG=ON" \
```

Comment:
Note: I wonder if we would not get some compatibility issues due to hardware optimized Cloudflare zlib, but I guess we wouldn't know unless we try! ;-)